UnicodeJsps/src/main/java/org/unicode/jsp/UBAVersion.java

```diff
@@ -0,0 +1,72 @@
+package org.unicode.jsp;
+
+import org.unicode.props.UcdPropertyValues.Age_Values;
+
+/**
+ * A class to encapsulate the available C UBA versions.
+ */
+public class UBAVersion {
+    static Age_Values[] C_UBA_AGES = {
```

Comment:
Please use `private` unless you really need & want the default package visibility.

Reply:
done